src/utils/currentUserCan.ts

```diff
@@ -10,4 +10,5 @@ const currentUserCan = (role: Role): ComputedRef<boolean> => {
 
 export default currentUserCan;
 
-export const canAdmin = currentUserCan(Role.Admin);
+export const canAdmin =
```

Comment:
I thought I had fixed this on the server side (i.e. the API should always return admin as well if superadmin), did you find it not working?

Reply:
Oh ok! I was looking into roles and noticed that function did not check for the SuperAdmin role, and thought it was an omission. We can throw this out if it's redundant